cpp/src/parquet/properties.h

```diff
@@ -245,6 +245,34 @@ class PARQUET_EXPORT ColumnProperties {
   bool page_index_enabled_;
 };
 
+// EXPERIMENTAL: Options for content-defined chunking.
+struct PARQUET_EXPORT CdcOptions {
+  /// Minimum chunk size in bytes, default is 256 KiB
```

Comment:
A chunk is always a data page, right? Can we explain this somewhere in these docstrings?

Reply:
Yes, adding it to the comment.

Reply:
> A chunk is always a data page, right?

IIUC, it may have many pages in a single chunk if the configured data page size is significantly smaller than the min chunk size. If yes, it would be much clearer if we explain the relationship between the chunk size and data page size configurations.

Reply:
Correct, but only if the max page size is smaller than the max_chunk_size. Also note that max page size counts encoded bytes while cdc max chunk size counts unencoded bytes.